.github/workflows/main.yml

```diff
@@ -29,6 +29,10 @@ jobs:
         run: |
           npm ci
 
+      - name: Run lint
+        run: |
+          npm run lint
```

Comment:
It would be cool if the package.json have a command to fix the errors.

Reply:
That's a great idea! Implemented.